amis/build_ami.sh

```diff
@@ -117,8 +117,9 @@ check_options() {
     available_arm_os="ubuntu1804 alinux2"  # subset of supported OSes for which ARM AMIs are available
     available_os="centos6 centos7 alinux ubuntu1604 ${available_arm_os}"
     cwd="$(dirname $0)"
-    tmp_dir=$(mktemp -d)
+    tmp_dir=${cwd}/../..
     export COOKBOOK_PATH="${tmp_dir}/cookbook"
+    export SCRIPT_PATH="${tmp_dir}/script"
```

Comment:
what happens if `SCRIPT_PATH` is not present? this is the case when `build_ami.sh` is called directly without passing from the `createami` command.

If we avoid to fail in case it's not present, we can omit the `"only":` attribute in the packer provisioners and always execute them